tmt/tests/booted/test-image-pushpull-upgrade.nu

```diff
@@ -65,6 +65,12 @@ RUN echo test content > /usr/share/blah.txt
     let progress = open --raw $progress_json | from json -o
     sanity_check_switch_progress_json $progress
 
+    # Check that /run/reboot-required exists and is a zero-byte file
+    let rr_meta = ls /run/reboot-required
+    assert equal $rr_meta.name "/run/reboot-required"
```

Comment:
I think this is failing because `ls` returns a table. It should work if you do `ls /run/reboot-required | first` instead.